uptane-standard.md

```diff
@@ -657,9 +657,9 @@ ECUs MUST have a secure source of time. An OEM/Uptane implementor MAY use any ex
 
 For an ECU to be capable of receiving Uptane-secured updates, it MUST have the following data provisioned at the time it is manufactured or installed in the vehicle:
 
-1. A sufficiently recent copy of required Uptane metadata at the time of manufacture or install. See the Uptane Deployment Considerations ({{DEPLOY}}) for more information.
-    * Partial verification ECUs MUST have the Root and Targets metadata from the Director repository.
-    * Full verification ECUs MUST have a complete set of metadata (Root, Targets, Snapshot, and Timestamp) from both repositories, as well as the repository mapping metadata ({{repo_mapping_meta}}).
+1. A sufficiently recent copy of required Uptane metadata at the time of manufacture or install. This is necessary for the ECU to authenticate that the remote repository is legitmate when it first downloads metadata in the field. See the Uptane Deployment Considerations ({{DEPLOY}}) for more information.
+    * Partial verification Secondary ECUs MUST have the Root and Targets metadata from the Director repository (to reduce the scope of rollback and replay attacks). These ECUs MAY also have metadata from other roles or the Image repository if they will be used by the Secondary.
+    * Full verification ECUs MUST have a complete set of metadata (Root, Targets, Snapshot, and Timestamp) from both repositories (to prevent rollback and replay attacks), as well as the repository mapping metadata ({{repo_mapping_meta}}). Delegations are not required.
```

Comment:
```suggestion
    * Full verification ECUs MUST have a complete set of metadata (Root, Targets, Snapshot, and Timestamp) from both repositories (to prevent all attacks covered by Uptane, including rollback and replay attacks), as well as the repository mapping metadata ({{repo_mapping_meta}}). Delegations are not required.
```

Reply:
I think we might be trying to address two separate concepts here. My goal here was to make it explicit why putting metadata into the device at the time of manufacture is important *as opposed to* just downloading it in the field and trusting on first use. To me, it sounds like this addition is trying to explain that, in general, you need to use all metadata to have all of the Uptane guarantees. I don't disagree with that, but I don't think this is the place to say that. Or have I misunderstood your intention?

I think this gets to a point that I've discussed recently with several colleagues: there are still many things in this Standard that are stated without providing a reason. Many times over the past few months, I've asked why it is recommended or required to do something, and ultimately someone has the reason in their head and they are able to explain it to me. In each case, I've tried to make a PR to lay out that logic directly in the text. Otherwise, we run the risk of this looking like a ransom set of requirements, and implementers may not have a clear picture of what the risks are of taking shortcuts or ignoring certain aspects.

Reply:
Ok, I think I misunderstood the goal here. In this case, the rollback and replay attacks are the most significant reason to list the metadata at installation time and it makes sense to call them out specifically.